chimedb/data_index/orm.py

```diff
@@ -411,6 +415,27 @@ class WeatherFileInfo(base_model):
     date = pw.CharField(null=True, max_length=8)
 
 
+class MiscFileInfo(base_model):
+    """Information about miscellaneous data tarballs.
+
+    Attributes
+    ----------
+    file : foreign key
+        Reference to the file this information is about.
+    start_time : float
+        Start of acquisition in UNIX time.
+    finish_time : float
+        End of acquisition in UNIX time.
+    data_type : string
+        The miscellaneous data type.
+    """
+
+    file = pw.ForeignKeyField(ArchiveFile, backref="miscinfos")
+    start_time = pw.DoubleField(null=True)
+    finish_time = pw.DoubleField(null=True)
+    data_type = pw.CharField()
```

Comment:
Given that these are all going to be fairly one off sorts of files, I wonder if we should be able to add extended descriptions of them in this table. It could be pulled out of the `metadata.json` file and be indexed in here.

If a file listing really is important it could get added into the extended description (see comment below).

Reply:
How about a JSON field with the rest of the metadata from the metadata file?

Reply:
Yeah. That sounds like a good idea to me.